docs/data/base/components/popper/popper.md

```diff
@@ -1,40 +1,52 @@
 ---
 product: base
-title: React Popper component
+title: Unstyled React Popper component
 components: PopperUnstyled
 githubLabel: 'component: Popper'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#tooltip
+packageName: '@mui/base'
 ---
 
 # Unstyled popper
 
-<p class="description">The Popper component lets you create components that are displayed on top of other elements.</p>
+<p class="description">The `PopperUnstyled` component lets you create tooltips and popovers that display information about an element on the page.</p>
 
-**Features**:
+`PopperUnstyled` relies on the third-party library ([Popper.js v2](https://popper.js.org/docs/v2/)) for perfect positioning.
 
-- 🕷 Popper relies on the 3rd party library ([Popper.js v2](https://popper.js.org/docs/v2/)) for perfect positioning.
-- 💄 It's an alternative API to [react-popper](https://popper.js.org/react-popper/v2/). It aims for simplicity.
-- The `anchorEl` is passed as the reference object to create a new `Popper.js` instance.
-- The children are placed in a [`Portal`](/base/react-portal/) prepended to the body of the document to avoid rendering problems.
-  You can disable this behavior with `disablePortal` prop.
-- The page scroll isn't blocked when the popper opens.
-- The placement of the popper updates with the available area in the viewport.
-
-**Caveats**:
-
-- Clicking away does not hide the `Popper` component. If you need this behavior, you can use [`ClickAwayListener`](/base/react-click-away-listener/)
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
-**Bundle size**: 📦 [8 kB gzipped](/size-snapshot).
+## Basic popper
 
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+```js
+import PopperUnstyled from '@mui/base/PopperUnstyled';
+```
 
-## Basic Popper
+By default, the popper is mounted to the DOM when its `open` prop is set to `true`, and removed from the DOM when `open` turns `false`.
 
-By default, the popper is mounted to the DOM when its `open` prop is set to `true` and removed from it when `open` turns `false`. If you want that to happen with a transition, use the `keepMounted` prop (see the [Transition](#transition) example below).
+`anchorEl` is passed as the reference object to create a new `Popper.js` instance.
+The children are placed in a [`Portal`](/base/react-portal/) prepended to the body of the document to avoid rendering problems.
+You can disable this behavior with `disablePortal` prop.
 
 {{"demo": "SimplePopper.js"}}
```

Comment:
Hi @samuelsycamore , I was going through the docs on the main page. And noticed that in the `SimplePopper` demo, the popper is not visible in dark mode - https://mui.com/base/react-popper/#basic-popper.

I think we need to simply remove the `backgroundColor: '#fff'` style applied in the child div for it to work in both modes. In dark mode, it overlaps with the white text.

Found this PR, so commenting here. Probably can be fixed in this PR.

Preview: https://deploy-preview-32412--material-ui.netlify.app/base/react-popper/#basic-popper

Reply:
@ZeeshanTamboli nice catch! I think we can open a dedicated PR for this, as this PR is mostly focused on the copy writing.